lib/MusicBrainz/Server/Report/ReleasesWithAmazonCoverArt.pm

```diff
@@ -0,0 +1,39 @@
+package MusicBrainz::Server::Report::ReleasesWithAmazonCoverArt;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ReleaseReport',
+     'MusicBrainz::Server::Report::FilterForEditor::ReleaseID';
+
+sub query {
+    "
+        SELECT 
+            r.id AS release_id,
+            row_number() OVER (ORDER BY ac.name COLLATE musicbrainz, r.name COLLATE musicbrainz)
+        FROM (
+            SELECT release_coverart.id
+            FROM release_coverart
+            WHERE cover_art_url ~ \'^https?://.*.images-amazon.com\'
```

Comment:
Those escapes look unnecessary (if harmless) because you use `"` for the quotes...but that probably also counts as an unnecessary interpolation, so maybe use `q()` too. :)
```suggestion
            WHERE cover_art_url ~ '^https?://.*.images-amazon.com'
```